### PR TITLE
German translation

### PR DESCRIPTION
**Pull request by Segdon**
## Pull request checklist

* [ ] I have tested my code
* [ ] I have read the [contributing guide](https://github.com/AnarchyLinux/installer/blob/HEAD/CONTRIBUTING.md)
* [ ] I have followed best practices and commented my code well

### CODE REVIEW

**Comment by erazemkokot** [comment]
Umm, although this wouldn't cause any problems, I won't merge it. Check out your commits. For every translation, you've submitted all the commits and all but the last one overwrite previous changes (although they don't change anything).

Before translating a new file (after submitting a pull request for the prevous file) you should delete your remove and local branch, then get the pushed changes from our pacman-fix branch and rebase them onto your version of pacman-fix and only then should you create a new branch from that. This way you only make a pull request with one commit (or multiple, just not the ones already pushed).

At this point I recommend you save the greek translation and any other files you changed somewhere outside the repo, delete the fork and refork the project, then follow the steps I outlined above, mainly
since the pacman-fix branch should be up-to-date after reforking, you should create a new branch from that.

Also, please use `git fetch` and `git rebase` instead of `git pull`, so you can avoid the "Merge remote-tracking branch..." commits.

I recommend you create a new project on your github account and practice these steps so you can see what I mean.

tl;dr

1. After making a pull request and us merging it you should delete the branch you made it on
2. Rebase our changes to pacman-fix to your pacman-fix branch
3. Create a new branch from your version of pacman-fix and make changes to it there
4. Make a pull request for that branch (so only the changes you made there get sent)
